modin/pandas/test/dataframe/test_default.py

```diff
@@ -585,20 +586,35 @@ def melt(df, *args, **kwargs):
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 @pytest.mark.parametrize(
-    "index", [lambda df: df.columns[0], lambda df: df[df.columns[0]].values, None]
+    "index",
+    [lambda df: df.columns[0], lambda df: df.columns[:2], lib.no_default],
+    ids=["one_column_index", "several_columns_index", "default"],
+)
+@pytest.mark.parametrize(
+    "columns", [lambda df: df.columns[len(df.columns) // 2]], ids=["one_column"]
```

Comment:
Why one column? We pick columns by len(df.columns) // 2?

Reply:
We take one column from the middle, there is no slice right there.

Reply:
Ah, I see. Missed that.